sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_new_profiles_v1/query.sql

```diff
@@ -1,26 +1,28 @@
 WITH pop AS (
   SELECT
-    ROW_NUMBER() OVER (PARTITION BY client_id) AS rn,
     client_id,
-    normalized_country_code AS country_code,
+    country AS country_code,
     normalized_channel AS channel,
-    application.build_id AS build_id,
+    app_build_id AS build_id,
     normalized_os AS os,
-    environment.settings.attribution.source AS attribution_source,
-    environment.partner.distribution_id AS distribution_id,
-    COALESCE(environment.settings.attribution.ua, '') AS attribution_ua,
-    DATE(submission_timestamp) AS date
+    mozfun.norm.truncate_version(normalized_os_version, "minor") AS os_version,
```

Comment:
So OS version was missing on this step, even though it was present in the activation step. I'm adding it here, but maybe there was a reason it was missing?